src/main/java/de/abstractolotl/azplace/AzplaceApplication.java

```diff
@@ -2,9 +2,16 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import redis.clients.jedis.Jedis;
 
 @SpringBootApplication
 public class AzplaceApplication {
+    @Bean
+    public Jedis jedis() {
+        Jedis jedis = new Jedis("192.168.1.242", 6379);
```

Comment:
Avoid HardCoded IPs and Ports

Reply:
Use @Value instead

Reply:
https://aztube.atlassian.net/browse/AX-106